_rules/inline-link-in-paragraph-has-distinguishable-content-e7cc2b.md

```diff
@@ -20,20 +18,17 @@ acknowledgements:
 
 ## Applicability
 
-This rule applies to any HTML element which:
+This rule applies to any [visible][] HTML element that is a [semantic link][], where the closest element (including itself) that generates an [inline box][] is part of a [line box][] that has [inline boxes][inline box] generated from an element containing [visible][] [text nodes][text node] that do not have a [semantic link][] as a [descendant][] in the [flat tree][].
```

Comment:
Same as in the other rules, I think more work needs to be done in this paragraph.